src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -55,11 +55,20 @@
     @Column(name = "inship_registration_end_date")
     private LocalDate registrationEndDate;
 
-//    private List<FormFullDTO> formList;
-//    private List<UserDTO> techList;
-//    private List<UserDTO> adminList;
-//    private List<LocationDTO> countryList;
-//    private InternshipFormat InternshipFormat;
-//    private List<Technology> technologyList;
+    @Column(name = "inship_format_name", nullable = false)
+    @Enumerated(EnumType.STRING)
+    @Type(type = "by.exadel.internship.mapper.enum_mapper.EnumTypePostgreSQL")
```

Comment:
why do you specify type here?

Reply:
it is for the future , when we will add some  Inrernship.

Reply:
we will think about how we could use it from one place, thanks